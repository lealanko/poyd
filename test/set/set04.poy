(* setting and verifying rooting *)
set(root:"10")
read("1.fas")
set(seed:1)
build(1)
report(trees)