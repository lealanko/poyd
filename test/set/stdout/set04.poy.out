(t1 (((t49 t25) (t28 (t27 ((t40 t33) ((t47 t38) ((t26 t18) (t20 (t6 t11))))))
)) (((t5 t4) (t30 t29)) ((t3 t14) ((t9 (t8 t32)) ((t41 (t2 t19)) (t48 (((t36
 t35) (t34 (t31 (t46 (t10 (t17 (t42 t39))))))) ((t7 t16) (t22 ((t12 (t43 (t45
 t44))) ((t15 (t24 t23)) (t13 (t37 t21))))))))))))));